### ISSUE TITLE
Questions on onset and frames  pretrained model  (maestro_checkpoint.zip)

### ISSUE THREAD

**Issue by nkjulia** [magenta/magenta#1920]
i clone the latest code and download the pretrained model ( maestro_checkpoint.zip)   to train my own model and did not change any model parameters.

i inspect the  pretrained checkpoint  and found there are 73087083 params  (# Total number of params: 73087083), but my new trained  model  has 67803243 params ( # Total number of params: 73087083).

when i use the eval mode using the checkpoint from  maestro_checkpoint.zip  , i got some errors ( onset and frames key frame/cudnn_lstm/stack_bidirectional_rnn/cell_0/bidirectional_rnn/bw/cudnn_compatible_lstm_cell/bias/adam not found in checkpoint)

i guess the model config maybe not the same, can anyone tell me the detail  ? thanks a lot.

**Comment by cghawthorne** [comment]
Hmm, that's definitely odd. I wonder if a new Tensorflow version has changed where the adam weights are expected to be stored. Can you provide a full stacktrace ?
